core/templates/services/page-title.service.ts

```diff
@@ -24,8 +24,10 @@ import { downgradeInjectable } from '@angular/upgrade/static';
   providedIn: 'root'
 })
 export class PageTitleService {
-  pageTitleForMobile: string = null;
-  pageSubtitleForMobile: string = null;
+  // The following properties are 'null' when no title needs to
+  // displayed in the mobile view.
+  pageTitleForMobile: string | null = null;
+  pageSubtitleForMobile: string | null = null;
```

Comment:
I think usage of `null` should be removed and we should return ` | undefined` at getters below

Reply:
hmmm, I think using `null` seems like a more cleaner approach.

Reply:
There are only a few places where the mobile subtitle is set like that of the topic editor and skill editor. (The name of the topic/skill is the subtitle)
![Screenshot from 2021-08-11 00-26-55](https://user-images.githubusercontent.com/43031234/128926385-4c919ba0-059e-470a-b78b-600a0907dda7.png)

At other places, like that of the exploration editor, the subtitle is not set and will be null there.
![Screenshot from 2021-08-11 00-52-13](https://user-images.githubusercontent.com/43031234/128926454-6c1dc21a-0c6a-4bb3-9c1a-cc1455e96d66.png)

Reply:
Hmm, okay, but I don't see the reason why this needs to be done in TS, why this cannot be handled just by CSS? Is there any case where the titles actually differ between mobile and non-mobily (except for the case where it is not shown)?

Reply:
Not sure to which exact file you're referring to in the linked commit.

But from my understanding, it is possible to do this in CSS and we can remove the Angular mobile titles? Can we do it? Feel free to create an issue and TODO for this.

Reply:
@vojtechjelinek on further inspection, the `setPageTitle()` was actually not setting the navbar titles of the pages, it was instead setting the document titles. (The `<title></title>` inside the html `<head></head>`)
The navbar or page titles are actually only set for mobile. (No page titles exist for non-mobily).
I have created a PR #13684 to rename the set/getPageTitle to set/getDocumentTitle to avoid confusion.

Reply:
Okay, moving with this to #13684